components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -151,5 +162,41 @@ export class ProjectEffects {
           message: `Could not update project: ${msg || payload.error}`
         });
       }));
+
+  @Effect()
+  applyRulesStart$ = this.actions$.pipe(
+    ofType<ApplyRulesStart>(ProjectActionTypes.APPLY_RULES_START),
+    mergeMap(() =>
+      this.requests.applyRulesStart().pipe(
+        switchMap(() => [
+          new ApplyRulesStartSuccess(),
```

Comment:
This is not currently hooked up to do anything. If that was deliberate, please at least add a comment here to the effect "needs to be wired up". Same goes for `ApplyRulesStartSuccess` below.